source/content/drupal-cms.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Drupal CMS on Pantheon
+description: Learn about installing and managing Drupal CMS sites on Pantheon.
+tags: [site, D11]
+permalink: docs/drupal-cms
+contributors: [alexmoreno, stevector]
+contenttype: [doc]
+innav: [true]
+categories: [create]
+cms: [drupal]
+audience: [development]
+product: [dashboard, terminus]
+integration: [--]
+reviewed: "2024-02-10"
+---
+
+# Drupal CMS
+
+Released in January of 2025, Drupal CMS is a relatively new wrapper around Drupal Core.
+Drupal CMS builds on the concept of [Recipes](https://www.drupal.org/docs/extending-drupal/drupal-recipes) to [radically reduce the amount of time and expertise required to stand up common functionality](https://pantheon.io/blog/drupal-cms-innovations).
+In particular, Drupal CMS is aimed at Marketers and Content Editors who commonly use Drupal, but who may not have the technical expertise to install and configure modules themselves.
+
+For Pantheon's developer-centric community, Drupal CMS will be most valuable as a reference point and proving ground for recipes.
+After trial usage of recipe combinations in sandbox sites, teams cab then replicate their favorite recipes on new or pre-existing live sites.
```

Comment:
This whole section feels like we're downselling Drupal CMS and giving reasons why it's not a real solution for Pantheon customers. I don't think that's the sense we want to portray. The suggestions I've made hopefully address this somewhat.

Reply:
I accepted those suggestions. What do you think now?